### DISCUSSION TITLE
Implement Email Notification for Password Resets

### DISCUSSION THREAD

**Discussion by coolcat-creations** [joomla/joomla-cms#44070] [2. Cold Features]
I would like to suggest adding a feature where users receive an email notification whenever their password is reset, whether initiated by the user or an admin. This is a common practice on many websites and serves as an important security and trust measure. It helps users stay informed about changes to their account and adds a layer of protection against unauthorized access.
What do you think about this?

**Comment by brianteeman** [comment]
Various extensions already provide this

**Comment by coolcat-creations** [comment]
I just searched but could not find any - but are you sure we speak of the same? I mean a mail to the user himself saying 
"Someone with the IP Adress xxx" just resetted your password... if that was not you contact the Administrator of the website.
Something like this?

**Reply by 4digitalworld** [reply]
This is a very important feature and I fully support it!

Also I think that the `User Sessions` feature should also be available in the **profile settings** for each user who logged into the website, similar to how it is done in social networks, when you can see a list of all your sessions on this website (example: Current Session/Windows, other sessions: Android, SmartTv, etc.)

currently only the administrator in the `Control Panel` can see all sessions of all users, but this should be available on the front end for each logged in user (the ability to view a list of all their sessions https://github.com/joomla/joomla-cms/discussions/44071).
